flexget/task.py

```diff
@@ -22,8 +21,10 @@
 from flexget.plugin import (
     DependencyError, get_plugins, phase_methods, plugin_schemas, PluginError, PluginWarning, task_phases)
 from flexget.utils import requests
+from flexget.utils.cached_input import get_config_hash
 from flexget.utils.database import with_session
 from flexget.utils.simple_persistence import SimpleTaskPersistence
+from flexget.utils.tools import merge_dict_from_to
```

Comment:
I know this isn't your name but can we change `merge_dict_from_to` to something like `dict_merge`? that name is horrible